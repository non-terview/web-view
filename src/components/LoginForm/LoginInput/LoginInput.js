import React from 'react';
import {useState} from "react";

function emailValidate(setEmail, email, event) {
    setEmail(event.target.value);
}

function passwordValidate(setEmail, email, event) {
    setEmail(event.target.value);
}

function LoginForm() {
    const [email, setEmail] = useState("");
    const [password, setPassword] = useState("");
    const [lang] = useState("ko");

    const langString =
        {
            ko: {
                email: "이메일", password: "비밀번호"
            },
            en: {
                email: "email", password: "password"
            }
        };


    return (
        <>
            <input type="text" placeholder={langString[lang].email} onChange={(event) =>
                emailValidate(setEmail, email, event)} value={email}>
            </input>
            <h1>{email}</h1>
            <input type="password" placeholder={langString[lang].password} onChange={(event =>
                passwordValidate(setPassword, email, event))}>
            </input>
            <h1>{password}</h1>

        </>
    );
}

export default LoginForm;
