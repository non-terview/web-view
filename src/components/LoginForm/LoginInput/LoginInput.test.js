import renderer from "react-test-renderer";
import React from "react";
import LoginInput from "./LoginInput";

test('LoginInput changes the class when hovered', () => {
        const component = renderer.create(
            <LoginInput/>,
        );

        let form = component.toJSON();

        expect(form).toMatchSnapshot();
    }
)