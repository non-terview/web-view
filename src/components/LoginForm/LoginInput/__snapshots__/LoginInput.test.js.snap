// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginInput changes the class when hovered 1`] = `
Array [
  <input
    onChange={[Function]}
    placeholder="이메일"
    type="text"
    value=""
  />,
  <h1>
    
  </h1>,
  <input
    onChange={[Function]}
    placeholder="비밀번호"
    type="password"
  />,
  <h1>
    
  </h1>,
]
`;
